packages/nbdime/package.json

```diff
@@ -23,22 +23,22 @@
     "watch": "tsc --build --watch"
   },
   "dependencies": {
-    "@jupyterlab/codeeditor": "^2.0.0",
-    "@jupyterlab/codemirror": "^2.0.0",
-    "@jupyterlab/coreutils": "^4.0.0",
-    "@jupyterlab/nbformat": "^2.0.0",
-    "@jupyterlab/outputarea": "^2.0.0",
-    "@jupyterlab/rendermime": "^2.0.0",
-    "@jupyterlab/services": "^5.0.0",
-    "@lumino/algorithm": "^1.1.2",
-    "@lumino/coreutils": "^1.3.0",
-    "@lumino/dragdrop": "^1.3.0",
-    "@lumino/signaling": "^1.2.2",
-    "@lumino/widgets": "^1.6.0",
+    "@jupyterlab/codeeditor": "^3.0.0",
+    "@jupyterlab/codemirror": "^3.0.0",
+    "@jupyterlab/coreutils": "^5.0.0",
+    "@jupyterlab/nbformat": "^3.0.0",
+    "@jupyterlab/outputarea": "^3.0.0",
+    "@jupyterlab/rendermime": "^3.0.0",
+    "@jupyterlab/services": "^6.0.0",
```

Comment:
Given that none of the JS code needed to change, do you think we should change the dependencies to `^5 || ^6` (and similar for other packages)? That would allow us to still support lab2 for now.

Reply:
I believe that would cause issues due to other dependancies like Lumino which need to run different versions depending on the lab version. Plus I would expect we would follow up this initial release with an update to pre-built extensions, which wouldn't be backward compatible. In general I think having user use nbdime 2.X for Lab 2.X and nbdime 3.X for lab 3.X make sense (to me)